projects/pgai/pgai/cli.py

```diff
@@ -119,28 +151,31 @@ async def run_workers(
     ),
     default="INFO",
 )
+@click.option(
+    "--poll-interval",
+    type=TimeDurationParamType(),
+    default="5m",
+    show_default=True,
+    help="The interval, in duration string or integer (seconds), to wait before checking for new work after processing all available work in the queue.",  # noqa
+)
+@click.option(
+    "--once",
+    type=click.BOOL,
+    default=False,
+    show_default=True,
+    help="Exit after processing all available work.",
+)
```

Comment:
If I want `--once`, it looks like I still have to provide a `--poll-interval` value.
Might be nice to make `--poll-interval` optional, OR make `--poll-interval=0` mean `--once`

Reply:
`--poll-interval` has a default , so you don't need to pass it always.

I thought about `--poll-interval=0` mean `--once`. My only issue is that it's not as explicit and maybe people don't realize that. But I'm willing to change it, I'm not 100% sold on either option really.

Reply:
okay, I'm fine with it as-is